_posts/2023-01-25-snowflake-snowpro-core-cert.md

```diff
@@ -0,0 +1,244 @@
+
+**GET_STAGE_LOCATION** -> Retrieves the URL for an external or internal named stage using the stage name as the input.
+
+**BUILD_SCOPED_FILE_URL** -> Generates a scoped Snowflake-hosted URL to a staged file using the stage name and relative file path as inputs. A scoped URL is encoded and permits access to a specified file for a limited period of time.
+
+**GET_PRESIGNED_URL** -> Generates the pre-signed URL to a staged file using the stage name and relative file path as inputs.
+
+**BUILD_STAGE_FILE_URL**-> Generates a Snowflake-hosted file URL to a staged file using the stage name and relative file path as inputs. A file URL permits prolonged access to a specified file. That is, the file URL does not expire.
+
+**GET_RELATIVE_PATH** -> Extracts the path of a staged file relative to its location in the stage using the stage name and absolute file path in cloud storage as inputs.
+
+### Differences between Snowflake functions of various types
+
+**TABLE FUNCTIONS** -> A table function returns a set of rows for each input row. The returned set can contain zero, one, or more rows. Each row can contain one or more columns.
+
+**SYSTEM FUNCTIONS** -> Snowflake provides the following types of system functions:
+
+- Control functions that allow you to execute actions in the system (e.g. aborting a query).
+- Information functions that return information about the system (e.g. calculating the clustering depth of a table).
+- Information functions that return information about queries (e.g. information about EXPLAIN plans).
+Many of these system functions have the prefix SYSTEM$ (e.g. SYSTEM\$TYPEOF).
+
+**WINDOW FUNCTIONS** -> A window function operates on a group (“window”) of related rows. Each time a window function is called, it is passed a row (the current row in the window) and the window of rows that contain the current row. The window function returns one output row for each input row. The output depends on the individual row passed to the function and the values of the other rows in the window passed to the function.
+
+**SCALAR FUNCTIONS** -> A scalar function is a function that returns one value per invocation; in most cases, you can think of this as returning one value per row. This contrasts with Aggregate Functions, which return one value per group of rows.
+
+**UDF FUNCTIONS** -> With user-defined functions (UDFs), you can extend the system to perform operations that are not available through the built-in, system-defined functions provided by Snowflake.
+
+**AGGREGATE FUNCTIONS** -> An aggregate function takes multiple rows (actually, zero, one, or more rows) as input and produces a single output. In contrast, scalar functions take one row as input and produce one row (one value) as output.
+
+### Snowflake built-in views and tables
+
+**AUTOMATIC_CLUSTERING_HISTORY** table -> Snowflake Information Schema:
+view the billing for Automatic Clustering
+
+**AUTOMATIC_CLUSTERING_HISTORY** View -> Account Usage:
```

Comment:
minor, mm bit of inconsistency with the View and view uppercase/lowercase spelling in this section as well as for REPLICATION_USAGE_HISTORY and REPLICATION_USAGE_HISTORY you have it as `Account Usage schema` and `Snowflake Information Schema`. But for the rest you got `INFORMATION_SCHEMA` and `Account Usage`